builtin/credential/aws/backend_test.go

```diff
@@ -1813,3 +1818,36 @@ func generateRenewRequest(s logical.Storage, auth *logical.Auth) *logical.Reques
 
 	return renewReq
 }
+
+func TestBackend_E2E_Initialize(t *testing.T) {
+
+	// All this test does is load up the aws plugin, so that if we are in
```

Comment:
This is basically making it a manual test, then. It's not actually testing what happened to the underlying storage entries.

Reply:
Yes we definitely need to make this a real test that forces it to actually upgrade some roles.

Reply:
I will add this post-beta